seqr/views/utils/pedigree_info_utils_2_3_tests.py

```diff
@@ -26,16 +28,18 @@ def test_parse_pedigree_table(self):
             [['family_id', 'individual_id', 'sex', 'affected', 'father', 'mother'],
              ['', '', 'male', 'u', '.', 'ind2']], FILENAME)
         self.assertListEqual(records, [])
-        self.assertListEqual(
-            errors, ["Error while converting {} rows to json: Family Id not specified in row #1:\n{{'affected': 'u', 'maternalId': 'ind2', 'individualId': '', 'sex': 'male', 'familyId': '', 'paternalId': ''}}".format(FILENAME)])
+        errors = errors[0].split('\n')
+        self.assertEqual(
+            errors[0], "Error while converting {} rows to json: Family Id not specified in row #1:".format(FILENAME))
         self.assertListEqual(warnings, [])
 
         records, errors, warnings = parse_pedigree_table(
             [['family_id', 'individual_id', 'sex', 'affected', 'father', 'mother'],
              ['fam1', '', 'male', 'u', '.', 'ind2']], FILENAME)
         self.assertListEqual(records, [])
-        self.assertListEqual(
-            errors, ["Error while converting {} rows to json: Individual Id not specified in row #1:\n{{'affected': 'u', 'maternalId': 'ind2', 'individualId': '', 'sex': 'male', 'familyId': 'fam1', 'paternalId': ''}}".format(FILENAME)])
+        self.assertIn(
+            errors[0], ["Error while converting {} rows to json: Individual Id not specified in row #1:\n{{'familyId': 'fam1', 'individualId': '', 'sex': 'male', 'affected': 'u', 'paternalId': '', 'maternalId': 'ind2'}}".format(FILENAME),
+                     "Error while converting {} rows to json: Individual Id not specified in row #1:\n{{u'affected': u'u', u'maternalId': u'ind2', u'individualId': u'', u'sex': u'male', u'familyId': u'fam1', u'paternalId': u''}}".format(FILENAME)])
```

Comment:
I like the way you tested the warning above by splitting on the newline better than the way you tested here with the 2 possible orders - can you do that here instead of this

Reply:
Updated.